test/js/pageobjects/CrossDeviceEverythingWeNeed.js

```diff
@@ -0,0 +1,28 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class CrossDeviceEverythingWeNeed extends Base {
```

Comment:
I think it will be easier for us to read and maintain the tests if these test classes have the same names of the component we are testing. For example, this component is called `CrossDeviceSubmit` so I would do  `class CrossDeviceSubmit extends Base {...}` here too

Reply:
Done.